test/runtests.jl

```diff
@@ -241,6 +240,24 @@ end
             @test 0.99 < sum(ϕ .* Δ)/amplitude < 1
         end
     end
+
+    @testset "hydrogen stark profiles" begin
```

Comment:
Minor suggestion: I would just add a brief comment clarifying where the test data comes from.

Maybe you could do something like the following?
```suggestion
    @testset "hydrogen stark profiles" begin
        # This test data was generated with Korg.hydrogen_line_absorption shortly
        # after writing the function. This data is consistent with the results
        # produced by the Fortran code distributed with Barklem+ 2000
```

Reply:
It's actually Stehle + 1999.  I've committed this comment but with the correct paper substituted.  (Barklem 2000 is the self-broadening profiles, for which we just use the standard ABO machinery).